pom.xml

```diff
@@ -146,7 +169,7 @@
           <version>3.0.0-M3</version>
           <executions>
             <execution>
-              <id>enforce-bytecode-version</id>
+              <id>enforce-bytecode-version</id> <!-- todo this configuration doesn't overwrite the execution defined in maven-parent -->
```

Comment:
It seems that this configuration doesn't overwrite the configuration with the same defined in maven-parent

Reply:
Fix see next commit